llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -582,6 +583,87 @@ static unsigned getMaxBitmapSize(const CounterMappingContext &Ctx,
   return MaxBitmapID + (SizeInBits / CHAR_BIT);
 }
 
+/// This holds the DecisionRegion and MCDCBranch(es) under it.
+/// Also traverses Expansion(s).
+struct DecisionRow {
+  /// The subject
```

Comment:
This comment is confusing

Reply:
It came from "should-name-everything-syndrome". I think it'd be enough to mention from the class comment.